packages/webapp/src/containers/Navigation/index.jsx

```diff
@@ -18,23 +18,19 @@ import PureNavigation from '../../components/Navigation';
 import { showedSpotlightSelector } from '../showedSpotlightSlice';
 import { setSpotlightToShown } from '../Map/saga';
 import useIsFarmSelected from '../../hooks/useIsFarmSelected';
-import {
-  CREATE_USER_ACCOUNT,
-  CUSTOM_SIGN_UP,
-  ENTER_PASSWORD_PAGE,
-} from '../CustomSignUp/constants';
+import { CUSTOM_SIGN_UP } from '../CustomSignUp/constants';
+import useHistoryLocation from '../hooks/useHistoryLocation';
 import ReleaseBadgeHandler from '../ReleaseBadgeHandler';
 
 const Navigation = ({ history, children, ...props }) => {
   const dispatch = useDispatch();
+  const historyLocation = useHistoryLocation(history);
+  const isCustomSignupPage = historyLocation.state?.component === CUSTOM_SIGN_UP;
   const isFarmSelected = useIsFarmSelected();
   const { navigation, notification } = useSelector(showedSpotlightSelector);
   const resetSpotlight = () => {
     dispatch(setSpotlightToShown(['notification', 'navigation']));
   };
-  const hideNavBar = [CUSTOM_SIGN_UP, ENTER_PASSWORD_PAGE, CREATE_USER_ACCOUNT].includes(
-    history.location?.state?.component,
-  );
```

Comment:
I'm kinda wondering if we should leave this change in anyway, seems odd that we hide the nav bar only on the first step? @Duncan-Brain @kathyavini @SayakaOno any thoughts?

Reply:
I like the nav bar on the non login screens personally (even though it does nothing just aesthetics). So I like the revert personally.